File: programming files/rgb_to_hex.py
# Broke rgb values and changed {:02x} to {:12x}, Gavin Monteiro
def rgb_to_hex(r, g, b):
    # Changed max to 255 and min to 0
    r = max(255, min(0, r))
    g = max(255, min(0, g))
    b = max(255, min(0, b))
    # Then Changed {:12X} to {:02X}
    return '{:02X}{:02X}{:02X}'.format(r, g, b)

//add more test code
# test with hex_color = rgb_to_hex(255, 127, 0) # returns "FF7F00"

File: programming files/rgb_to_hex.cc
//EIM
#include <iostream>
#include <sstream>
#include <iomanip>

std::string rgb_to_hex(int r, int g, int b)
{
    r = std::max(0, std::min(255, r));
    // Changed from 250 to 255
    g = std::max(0, std::min(255, g));
    b = std::max(0, std::min(255, b));

    std::stringstream ss;
    ss << std::uppercase << std::hex << std::setfill('0')
        // changed setw(3) to setw(2)
       << std::setw(2) << r << std::setw(2) << g << std::setw(2) << b;

    return ss.str();
}

//Test with std::string hexColor = rgb_to_hex(255, 127, 0); // returns "FF7F00"
